### PR TITLE
[storybook__addon-knobs] Update for v5

### PR DESCRIPTION
**Pull request by azmenak**
Updating for changes made in v5

https://github.com/storybooks/storybook/tree/next/addons/knobs

### CODE REVIEW

**Review by joscha — COMMENTED** [review]
I think for v5 there should be types coming with the package, are they not?

**Comment by azmenak** [comment]
> I think for v5 there should be types coming with the package, are they not?

@joscha I don't think so, using `@storybook/addon-knobs@5.0.8`

I'm seeing this in VSCode

Screen Shot 2019-04-15 at 9 25 52 AM

Also checked the files installed in `node_modules`, no type definitions in there

File: types/storybook__addon-knobs/index.d.ts [hunk 1]
```diff
@@ -65,6 +66,25 @@ export function array<T>(name: string, value: ReadonlyArray<T>, separator?: stri
...
+    | "check"
+    | "inline-check"
+    | "select"
+    | "multi-select";
+}
+
+export function optionsKnob<T>(
+    label: string,
+    values: {
+        [key: string]: any;
```
Comment (joscha) [inline types/storybook__addon-knobs/index.d.ts]:
```suggestion
        [key: string]: T;
```

File: types/storybook__addon-knobs/index.d.ts [hunk 2]
```diff
@@ -65,6 +66,25 @@ export function array<T>(name: string, value: ReadonlyArray<T>, separator?: stri
...
+    },
+    defaultValue?: T,
```
Comment (joscha) [inline types/storybook__addon-knobs/index.d.ts]: any chance for us to make sure `defaultValue` is a value in `values`?

**Review by joscha — CHANGES_REQUESTED** [review]
Intent LGTM, I think the values should follow the generic. The defaultValue would be an added bonus but I think it would require changes in storybook itself.

**Review by joscha — APPROVED** [review]

**Pull request merged**